app/config/locale/templates/email-base-branded.tpl

```diff
@@ -0,0 +1,98 @@
+<!doctype html>
+<html>
+    <head>
+        <link
+            href="https://fonts.googleapis.com/css2?family=Inter:wght@400;500;600&family=Poppins:wght@500;600&display=swap"
+            rel="stylesheet"
+        />
+        <style>
+            .main {
+                padding: 32px;
+                color: #616b7c;
+                font-size: 15px;
+                font-family: "Inter", sans-serif;
+            }
+
+            table {
+                width: 100%;
+                border-spacing: 0 !important;
+            }
+
+            table,
+            tr,
+            th,
+            td {
+                margin: 0;
+                padding: 0;
+            }
+
+            td {
+                vertical-align: top;
+            }
+
+            .main a {
+                color: currentColor;
+            }
+
+            h1,
+            h2,
+            h3,
+            h4,
+            h5,
+            h6 {
+                font-family: "Poppins", sans-serif;
+            }
+
+            hr {
+                border: none;
+                border-top: 1px solid #e8e9f0;
+            }
+
+            p {
+                margin-bottom: 10px;
+            }
+        </style>
+    </head>
+
+    <body>
+        <div
+            class="main"
+            style="
+                direction: {{direction}};
+                max-width: 650px;
+                word-wrap: break-word;
+                overflow-wrap: break-word;
+                word-break: break-all;
```

Comment:
Please confirm with Torsten or Elad about this. Sara didn't like screenshots I was sending her, it was breaking words in the middle.

What I did instead was `break-word`, and only set `break-all` on links (`a`)

Reply:
I agree, `break-word` wasn't looking good. We should use `break-all`

Reply:
Not sure if we are on same page.

`word-break: break-all;` will break word in the middle
`word-break: break-word;` will only break after a word

I propose to use second as default, and only use first for links, as they are long and unreadable anyway.

Reply:
Oh, sorry. Yeah, that's what I mean too.

Reply:
No strong opinion, whatever design think is best.

Reply:
Sara already didnt like it when reviewing my MagicURL email with this issue, you can go ahead and apply what I mentioned.